### DISCUSSION TITLE
Webdev不能被登录，老显示file sync settings错误。

### DISCUSSION THREAD

**Discussion by liumwei** [cloudreve/cloudreve#1393] [General]
登录cloudreve Webdev服务，但老出现如下错误：

The WebDAV server did not accept the username and password you entered.

Please check your file sync settings or contact your WebDAV server administrator.

确认cloudreve已经开启了Webdev服务，账户和密码没有输入错误。这是哪里的原因呢？

**Comment by liumwei** [comment]
这个file sync settings是在哪里设置？如何设置?

**Reply by HFO4** [reply]
密码不是 Cloudreve 账户密码。
